src/hooks/useSubStep/index.ts

```diff
@@ -0,0 +1,51 @@
+import {useCallback, useRef, useState} from 'react';
+import type {SubStepProps, UseSubStep} from './types';
+
+export default function useSubStep<TProps extends SubStepProps>({bodyContent, onFinished, startFrom = 0}: UseSubStep<TProps>) {
```

Comment:
Maybe some generic docs for this hook would be useful in sense of what does this control exactly.

Reply:
added a basic description from the doc on this flow, lmk if you'd like it altered in some way